src/Microsoft.Azure.ServiceBus/Amqp/AmqpMessageSender.cs

```diff
@@ -65,6 +86,75 @@ public override Task CloseAsync()
             }
         }
 
+        protected override async Task<long> OnScheduleMessageAsync(BrokeredMessage brokeredMessage)
+        {
+            // TODO: Ensure System.Transactions.Transaction.Current is null. Transactions are not supported by 1.0.0 version of dotnet core.
+            TimeoutHelper timeoutHelper = new TimeoutHelper(this.OperationTimeout, true);
+            using (AmqpMessage amqpMessage = AmqpMessageConverter.ClientGetMessage(brokeredMessage))
+            {
+                RequestResponseAmqpLink amqpLink =
+                    await
+                        this.RequestResponseLinkManager.GetOrCreateAsync(timeoutHelper.RemainingTime())
+                            .ConfigureAwait(false);
+
+                var request = AmqpRequestMessage.CreateRequest(
+                    ManagementConstants.Operations.ScheduleMessageOperation,
+                    this.OperationTimeout,
+                    null);
+
+                ArraySegment<byte>[] payload = amqpMessage.GetPayload();
+                BufferListStream buffer = new BufferListStream(payload);
+                ArraySegment<byte> value = buffer.ReadBytes((int)buffer.Length);
+
+                var entry = new AmqpMap();
+                {
+                    entry[ManagementConstants.Properties.Message] = value;
+                    entry[ManagementConstants.Properties.MessageId] = brokeredMessage.MessageId;
+
+                    if (!string.IsNullOrWhiteSpace(brokeredMessage.SessionId))
+                    {
+                        entry[ManagementConstants.Properties.SessionId] = brokeredMessage.SessionId;
+                    }
+
+                    if (!string.IsNullOrWhiteSpace(brokeredMessage.PartitionKey))
+                    {
+                        entry[ManagementConstants.Properties.PartitionKey] = brokeredMessage.PartitionKey;
+                    }
+                }
+
+                request.Map[ManagementConstants.Properties.Messages] = new List<AmqpMap> { entry };
+
+                IEnumerable<long> sequenceNumbers = null;
+                var response = await this.ExecuteRequestResponseAsync(request);
+                if (response.StatusCode == AmqpResponseStatusCode.OK)
+                {
+                    sequenceNumbers = response.GetValue<long[]>(ManagementConstants.Properties.SequenceNumbers);
+                }
+
+                return sequenceNumbers?.FirstOrDefault() ?? 0;
```

Comment:
0 is not a valid sequence number. I've written it as such to keep it consistent with the other published library. I'm working on a task for exception handling in this path. Will send out a separate PR for that... Leaving it as is for this PR.